exporter/exportertest/nop_exporter_test.go

```diff
@@ -21,19 +21,19 @@ import (
 	metricspb "github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1"
 	tracepb "github.com/census-instrumentation/opencensus-proto/gen-go/trace/v1"
 	"github.com/open-telemetry/opentelemetry-service/consumer/consumerdata"
+	"github.com/stretchr/testify/require"
```

Comment:
nit: follow the imports grouping pattern (1st: standard lib; 2nd: third-parties; 3rd: otel-svc)